src/vm/ceeload.cpp

```diff
@@ -192,6 +192,35 @@ BOOL Module::SetTransientFlagInterlocked(DWORD dwFlag)
 }
 
 #if PROFILING_SUPPORTED 
+void Module::UpdateNewlyAddedTypes()
```

Comment:
You should also open an issue in the docs repo to track updating the docs to match the new behavior.

Reply:
Opened https://github.com/dotnet/docs/pull/10510